generators/gulp/templates/gulpfile.js

```diff
@@ -1,12 +1,13 @@
+<% if (!babel) { -%>
```

Comment:
Same error https://github.com/yeoman/generator-node/pull/151#discussion_r35397141

gulpfile should be written in ES2015 if babel ?